bin/run-prod.sh

```diff
@@ -4,12 +4,11 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-function run-gunicorn () {
+function run-uwsgi () {
     if [[ -z "$NEW_RELIC_LICENSE_KEY" ]]; then
-        exec gunicorn "$@"
+        exec uwsgi "$@"
     else
-        export NEW_RELIC_CONFIG_FILE=newrelic.ini
-        exec newrelic-admin run-program gunicorn "$@"
+        NEW_RELIC_CONFIG_FILE=newrelic.ini newrelic-admin run-program uwsgi "$@"
```

Comment:
I think we need the exec call here so that the docker agent is attached to the right process

Reply:
Good point, we don't need `run-prod.sh` as the process and can replace it with the newrelic + uwsgi process. I'll update it back to the original form. I'm doing this in basket so maybe I'll update it there as well so things are the same.

Reply:
Updated